runtime/acala/src/lib.rs

```diff
@@ -1429,6 +1442,40 @@ impl TakeRevenue for ToTreasury {
 	}
 }
 
+pub struct AcalaDropAssets<X, T>(PhantomData<(X, T)>);
+impl<X, T> DropAssets for AcalaDropAssets<X, T>
+where
+	X: DropAssets,
+	T: TakeRevenue,
+{
+	fn drop_assets(origin: &MultiLocation, assets: Assets) -> Weight {
+		let multi_assets: Vec<MultiAsset> = assets.into();
+		let mut asset_traps: Vec<MultiAsset> = vec![];
+		for asset in multi_assets {
+			if let MultiAsset {
+				id: Concrete(location),
+				fun: Fungible(amount),
+			} = asset.clone()
+			{
+				let currency_id = CurrencyIdConvert::convert(location);
+				// burn asset(do nothing here) if convert result is None
+				if let Some(currency_id) = currency_id {
```

Comment:
when convert result is None, do nothing has the meaning of burn, right?

Reply:
yes. we just burn unknown assets for now. later maybe we can have them in the unknown-tokens pallet